configs/_base_/scheduler/median.py

```diff
@@ -0,0 +1,9 @@
+# Refer to https://github.com/ray-project/ray/blob/master/python/ray/tune/schedulers/median_stopping_rule.py  # noqa
+
+trial_scheduler = dict(
+    type='MedianStoppingRule',
+    time_attr='time_total_s',
+    grace_period=60,
```

Comment:
Can I ask you to add a UT?

Reply:
I'll make test codes for all schedulers in the next PR.